Assets/ZSSRichTextEditor.js

```diff
@@ -274,8 +272,12 @@ ZSSEditor.getSelectedText = function() {
 ZSSEditor.getCaretArguments = function() {
     var caretInfo = this.getYCaretInfo();
     
-    this.caretArguments[0] = 'yOffset=' + caretInfo.y;
-    this.caretArguments[1] = 'height=' + caretInfo.height;
+    if (caretInfo == null) {
+        this.caretArguments = null;
```

Comment:
Good eye.  It didn't break while I was testing but you're right that it will after being set to null once.